wiki/Achievements/en.md

```diff
@@ -94,12 +98,13 @@ Dedication
 | ![25,000 Plays (osu! mode)](/wiki/Achievements/img/osu-plays-25000.png "25,000 Plays (osu! mode)") | ![3,000,000 Drum Hits](/wiki/Achievements/img/taiko-hits-3000000.png "3,000,000 Drum Hits") | ![Catch 2,000,000 fruits](/wiki/Achievements/img/fruits-hits-2000000.png "Catch 2,000,000 fruits") | ![4,000,000 Keys](/wiki/Achievements/img/mania-hits-4000000.png "4,000,000 Keys") |
 | ![50,000 Plays (osu! mode)](/wiki/Achievements/img/osu-plays-50000.png "50,000 Plays (osu! mode)") | ![30,000,000 Drum Hits](/wiki/Achievements/img/taiko-hits-30000000.png "30,000,000 Drum Hits") | ![Catch 20,000,000 fruits](/wiki/Achievements/img/fruits-hits-20000000.png "Catch 20,000,000 fruits") | ![40,000,000 Keys](/wiki/Achievements/img/mania-hits-40000000.png "40,000,000 Keys") |
 
-
 Mod Introduction
------------------
+----------------
 
-**The Mod Introduction** achievements can be obtained by clearing maps with the specified game modifier. They're meant to encourage newer players to try out each mod and explore what the game has to offer.
+_See also: [Game Modifiers](/wiki/Game_Modifiers)._
 
+The Mod Introduction achievements can be obtained by clearing maps with the specified game modifier.
+They are meant to encourage newer players to try out each mod and explore what the game has to offer.
```

Comment:
will use that; I won't rephrase because I will use `you` in the rephrasing and you will nitpick